### PR TITLE
fix: add missing clean up for instance segmentation docker container

### PR DESCRIPTION
**Pull request by jim-wang-intel**
add make clean up segmentation docker container for ovms-client clean up

Fixes: #215

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**215**

### CODE REVIEW

**Review by vli11 — COMMENTED** [review]
do you want to add clean-segmentation on .PHONY?

**Comment by jim-wang-intel** [comment]
> do you want to add clean-segmentation on .PHONY?

yes, I can, thanks

**Review by vli11 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -38,12 +40,15 @@ get-server-code:
 	echo "Getting model_server code"
 	git clone https://github.com/gsilva2016/model_server 
 
-clean-ovms-client: clean-grpc-go
+clean-ovms-client: clean-grpc-go clean-segmentation
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply Makefile:43]: yes, good catch

[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply Makefile:43]: yes, the segmentation is launched by ovms-client. if ovms-client is cleaned up; we should also clean up the segmentation; just like grpc-go

**Review by antoniomtz — APPROVED** [review]
LGTM

**Pull request merged**